src/Util/PHP/Template/TestCaseMethod.tpl.dist

```diff
@@ -47,7 +47,12 @@ function __phpunit_run_isolated_test()
     $result->beStrictAboutTodoAnnotatedTests({isStrictAboutTodoAnnotatedTests});
     $result->beStrictAboutResourceUsageDuringSmallTests({isStrictAboutResourceUsageDuringSmallTests});
 
-    $test = new {className}('{methodName}', unserialize('{data}'), '{dataName}');
```

Comment:
Of course, I think so, especially that I've just made changes to TestCaseMethod file.